files/build_templates/sonic_debian_extension.j2

```diff
@@ -328,7 +328,7 @@ sudo cp $IMAGE_CONFIGS/monit/restart_service $FILESYSTEM_ROOT/usr/bin/
 sudo chmod 755 $FILESYSTEM_ROOT/usr/bin/restart_service
 
 # Install custom-built smartmontools
-sudo dpkg --root=$FILESYSTEM_ROOT -i $debs_path/smartmontools_*.deb
+sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y install smartmontools 7.2-1
```

Comment:
This is going to cause apt to grep for `7.2-1` in its entire package cache...

I think you meant `apt-get -y install smartmontools=7.2-1`

Reply:
fixed.